contrib/tracing/Dockerfile

```diff
@@ -0,0 +1,75 @@
+#for kernel libraries
+FROM linuxkit/kernel:4.15 as ksrc
+
```

Comment:
Since we can't test these tracepoints on Macos, I created this container with tapyrus and usdt packages. I use this container to build and test the tracepoints. visual studio code docker integration makes coding and debugging inside the container so convenient. It need not be in the repository if I'm the only person using it though.

Reply:
If you change `Dockerfile` in the root dir, wouldn't this be necessary?

Reply:
Yes, this works! I was able to combine the two files. Now build, test and trace are all available inside the tapyrus container.

Reply:
I verified that the container can build successfully. But didn't check if bpftrace or bcc can be run inside it.